lslidar_n301_decoder_node:
  ros__parameters:
    child_frame_id: laser_link              
    point_num: 2000            
    angle_disable_min: 0.0
    angle_disable_max: 0.0
    min_range: 0.3
    max_range: 100.0 
    frequency: 10.0  
    use_gps_ts: false
    gps_correct: true
    publish_point_cloud: true
    filter_scan_point: true

    
lslidar_n301_driver_node:
  ros__parameters:
    frame_id: laser_link
    device_ip: 192.168.1.222
    device_port: 2368
    add_multicast: false
    group_ip: 224.1.1.2

    
